File: Core/Model/Item.cs
using System;

namespace Core.Model
{
    public class Item
    {
        public virtual Guid Id { get; protected set; }
        public virtual string Title { get; protected set; }
        public virtual string Creator { get; protected set; }
        public virtual string Edition { get; protected set; }

        protected Item() { }
    }
}

File: Lending.Web/App/models/Item.js
var attr = DS.attr;
App.Item = DS.Model.extend({
    //id: attr('string'),
    title: attr('string'),
    creator: attr('string'),
    edition: attr('string'),

    //userOwnership: DS.belongsTo('UserOwnership'),

    hasError: function () {
        var currentError = this.get("error");
        return !(currentError === '' || currentError == null);
    }.property('error'),

});
